tests/installation/agama_reboot.pm

```diff
@@ -70,14 +70,16 @@ sub agama_system_prepare_ipmi {
     record_info("Device information", "Device: ${device}\nRoot partition: ${root_partition}");
     assert_script_run("mount ${root_partition} /mnt");
     # Set correct serial console to be able to see login in first boot
-    record_info("Set serial console");
-    my $sol_console = (is_aarch64) ? get_var('SERIALCONSOLE', 'ttyAMA0') : get_var('SERIALCONSOLE', 'ttyS1');
-    assert_script_run("sed -i 's/quiet/console=$sol_console,115200/g' /mnt/boot/grub2/grub.cfg");
-    # Upload original grub configuration
-    upload_logs("/mnt/etc/default/grub", failok => 1);
-    upload_logs("/mnt/boot/grub2/grub.cfg", failok => 1);
-    # Set permanent grub configuration
-    assert_script_run("sed -i 's/quiet/console=$sol_console,115200/g' /mnt/etc/default/grub");
+    if (is_ipmi) {
```

Comment:
We may need handle serial console, e.g. `console=hvc0` for ppc64le

Reply:
Thanks, let me try with build 3.21.

Reply:
BTW, the workaround mentioned above will be removed totally once `post scripts` works fine in agama auto config files

Reply:
fixed

Reply:
VR: https://openqa.suse.de/tests/16288057

https://openqa.suse.de/tests/16288057/file/agama_reboot-grub